madmom/evaluation/key.py

```diff
@@ -175,29 +301,85 @@ class KeyMeanEvaluation(EvaluationMixin):
         ('correct', 'Correct'),
         ('fifth', 'Fifth'),
         ('relative', 'Relative'),
+        ('relative_of_fifth', 'Relative of Fifth'),
         ('parallel', 'Parallel'),
         ('other', 'Other'),
         ('weighted', 'Weighted'),
     ]
 
     def __init__(self, eval_objects, name=None):
         self.name = name or 'mean for {:d} files'.format(len(eval_objects))
-
+        self.relative_of_fifth_present = False
+        if check_key_eval_objects(eval_objects):
+            self._count_evaluations(eval_objects)
+            if self.relative_of_fifth > 0:
+                self.relative_of_fifth_present = True
+        else:
+            raise ValueError('The KeyEvaluation objects are not '
+                             'all the same.')
+
+    def _count_evaluations(self, eval_objects):
         n = len(eval_objects)
         c = Counter(e.error_category for e in eval_objects)
-
         self.correct = float(c['correct']) / n
         self.fifth = float(c['fifth']) / n
         self.relative = float(c['relative']) / n
         self.parallel = float(c['parallel']) / n
         self.other = float(c['other']) / n
+        self.relative_of_fifth = float(c['relative_of_fifth']) / n
         self.weighted = sum(e.score for e in eval_objects) / n
 
     def tostring(self, **kwargs):
-        return ('{}\n  Weighted: {:.3f}  Correct: {:.3f}  Fifth: {:.3f}  '
-                'Relative: {:.3f}  Parallel: {:.3f}  Other: {:.3f}'.format(
-                    self.name, self.weighted, self.correct, self.fifth,
-                    self.relative, self.parallel, self.other))
+        ret = ''
+        spacing = ' ' * 2
+        ret += '{}\n'.format(self.name) + spacing
+        ret += 'Weighted: {:.3f}'.format(self.weighted) + spacing
+        ret += 'Correct: {:.3f}'.format(self.correct) + spacing
+        ret += 'Fifth: {:.3f}'.format(self.fifth) + spacing
+        ret += 'Relative: {:.3f}'.format(self.relative) + spacing
+        if self.relative_of_fifth_present:
+            ret += 'Relative of fifth: {:.3f}'.format(self.relative_of_fifth) \
+                   + spacing
+        ret += 'Parallel: {:.3f}'.format(self.parallel) + spacing
+        ret += 'Other: {:.3f}'.format(self.other)
+        return ret
+
+
+def check_key_eval_objects(key_eval_objects):
```

Comment:
I feel this function and the ones below can be simplified a lot by doing something like this (not tested!):
```python
def _check_key_eval_objects(key_eval_objects):
    e = key_eval_objects[0]
    for ke in key_eval_objects[1:]:
        if (ke.error_score != e.error_scores or
            ke.strict_fifth != e.strict_fifth or
            ke.relative_of_fifth != e.relative_of_fifth):
            return False	
    return True
```
Can you check if this is equivalent and if yes, use the simpler code?
Also, I do not think this function makes sense outside of this module, so we should mark it as "private" by using the underscore.

Reply:
Indeed, this simpler version works fine. Thanks !